scenarios/kubernetes_e2e_test.py

```diff
@@ -28,6 +28,17 @@
 
 import kubernetes_e2e
 
+FAKE_SCM = 'STABLE_BUILD_GIT_COMMIT 599539dc0b99976fda0f326f4ce47e93ec07217c\n' \
```

Comment:
Minor nitpick: There's a lot of information in here besides the SCM, so calling it `FAKE_WORKSPACE_STATUS` seems more appropriate. Same goes for `fake_output_scm` -> `fake_output_workspace_status`.